package.json

```diff
@@ -10,12 +10,19 @@
   },
   "dependencies": {
     "autoprefixer": "10.4.14",
+    "axios": "^1.4.0",
+    "classnames": "^2.2.6",
     "eslint": "8.46.0",
     "eslint-config-next": "13.4.13",
+    "eventsource-parser": "^1.0.0",
     "next": "13.4.13",
+    "openai": "^3.3.0",
     "postcss": "8.4.27",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-fireworks": "^1.0.4",
+    "react-icons": "^4.10.1",
+    "sass": "^1.65.1",
```

Comment:
are you using sass?

Reply:
I'm using only react-icons in the prompt file
(import { FaSmile, FaSadTear, FaSpinner } from "react-icons/fa";)

will remove the unnecessary packages